react/features/settings/components/web/MoreTab.js

```diff
@@ -220,6 +231,26 @@ class MoreTab extends AbstractDialogTab<Props, State> {
 
     _onKeyboardShortcutEnableChanged: (Object) => void;
 
+    /**
+     * Callback invoked to select if the given type of
+     * notifications should be shown.
+     *
+     * @param {Object} e - The key event to handle.
+     * @param {string} type - The type of the notification.
+     *
+     * @returns {void}
+     */
+    _onNotificationsChanged({ target: { checked } }, type) {
+        super._onChange({
+            notifications: {
```

Comment:
I would also rename this to something else, either also `enabledNotifications` or some other different name.